assign_rights/templates/users/login.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="row justify-content-sm-center">
+  <div class="card col-sm-6 p-4">
+    <form action="." method="post">
+      {{ form.non_field_errors }}
+
+      {% csrf_token %}
+
+      <div class="form-group has-feedback">
+        {{ form.username.errors }}
+        <input type="text" name="username" id="{{ form.username.id_for_label }}" class="form-control" placeholder="Username" value="{% if form.username.value %}{{ form.username.value }}{% endif %}" />
```

Comment:
Both inputs in the form need associated form labels where the "for" value matches the input id: `<label for="id_for_label">Username</label>`. The placeholder text is not necessary/not optimal. See https://webaim.org/techniques/forms/controls